app/controllers/companies_controller.rb

```diff
@@ -13,7 +15,9 @@ def update
     redirect_to company_path(@company)
   end
 
-  def show; end
+  def show
+    @company_profile = @company.company_profile
```

Comment:
Acho que a rota de show não é tão útil pra isso por causa do ID, você não precisa do ID para montar essa view ou não? O que acha?

Reply:
Essa parte deve ficar assim pois qualquer pessoa pode conseguir ver o perfil de qualquer empresa. Os botões de edição que não podem ser visualizados.

Reply:
@laurameireles23 o que você quiz dizer com qualquer pessoa? se for qualquer usuário o `before_action :authenticate_employee!` faz com que isso não aconteça

Reply:
Resolvido :)